app/helpers/bets_helper.rb

```diff
@@ -0,0 +1,9 @@
+module BetsHelper
+  def creator?
+    current_user == @bet.creator
+  end
+
+  def resolved?
```

Comment:
Generally, I don't think an helper is warranted in this case. The helpers should be instance methods on the Bet model